alerting/src/utils/server-utils.ts

```diff
@@ -2,6 +2,7 @@ import fetch from 'node-fetch';
 import moment from 'moment';
 import { xml2js } from 'xml-js';
 import { get, isEmpty, isString, union } from 'lodash';
+import { formatUrl } from "../../../packages/common";
```

Comment:
I would expect this import to be something like:

`import { formatUrl } from "common";`

Not a blocker for this PR

Reply:
okay.  I'll look into this.